tests/test_study.py

```diff
@@ -498,16 +503,31 @@ def f(trial):
             assert df.value[i] == 3.5
             assert isinstance(df.datetime_start[i], pd.Timestamp)
             assert isinstance(df.datetime_complete[i], pd.Timestamp)
-            assert df.params.x[i] == 1
-            assert df.params.y[i] == 2.5
-            assert df.user_attrs.train_loss[i] == 3
-            assert df.system_attrs._number[i] == i
-            if include_internal_fields:
-                assert ('distributions', 'x') in df.columns
-                assert ('distributions', 'y') in df.columns
-                assert ('trial_id', '') in df.columns  # trial_id depends on other tests.
-                assert ('distributions', 'x') in df.columns
-                assert ('distributions', 'y') in df.columns
+
+            if multi_index:
+                if include_internal_fields:
+                    assert ('distributions', 'x') in df.columns
+                    assert ('distributions', 'y') in df.columns
+                    assert ('trial_id', '') in df.columns  # trial_id depends on other tests.
+                    assert ('distributions', 'x') in df.columns
+                    assert ('distributions', 'y') in df.columns
+
+                assert df.params.x[i] == 1
+                assert df.params.y[i] == 2.5
+                assert df.user_attrs.train_loss[i] == 3
+                assert df.system_attrs._number[i] == i
+            else:
+                if include_internal_fields:
+                    assert 'distributions_x' in df.columns
+                    assert 'distributions_y' in df.columns
+                    assert 'trial_id' in df.columns  # trial_id depends on other tests.
+                    assert 'distributions_x' in df.columns
```

Comment:
These two lines look redundant because L524 and L525 are exactly the same with L521 and L522 respectively.

Reply:
Thanks for the catch. Fixed accordingly and the checks above as well which were duplicated too.